node/module_all.ts

```diff
@@ -118,6 +119,15 @@ export default {
   path,
   "perf_hooks": perfHooks,
   process,
+  get punycode() {
+    process.emitWarning(
```

Comment:
Lazily prints the deprecation warning because this message should be shown only when the punycode module is required.